cpp/test/Ice/proxy/AllTests.cpp

```diff
@@ -417,21 +417,24 @@ allTests(TestHelper* helper)
     //
     // Test proxies with duplicated endpoint option.
     //
-    string goodBase = "hello:default -h host1 -p 4061 --sourceAddress 127.0.0.1";
-    test(communicator->stringToProxy(goodBase));
+    if (protocol != "bt")
```

Comment:
In Java we use indexOf("bt") which also match "bts". In theory you can set "--protocol=bts" in the command line